core-service/src/test/java/com/crmfoodestablishment/coreservice/web/GlobalExceptionHandlerTest.java

```diff
@@ -0,0 +1,70 @@
+package com.crmfoodestablishment.coreservice.web;
+
+import com.crmfoodestablishment.coreservice.service.exception.NotFoundException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.FieldError;
+import org.springframework.validation.ObjectError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.when;
+
+public class GlobalExceptionHandlerTest {
```

Comment:
Global exception handler shouldn't be tested in this way
exception handling should be a part of integration 'mockMvc' test

Reply:
refactored this test, now handler a part of integration 'mockMvc' test.